weather.ozweather/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="weather.ozweather" name="Oz Weather" version="1.2.0" provider-name="Bossanova808">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.pil" version="1.1.7"/>
```

Comment:
Well, I can't see it the module here - https://github.com/xbmc/repo-scripts/tree/matrix ?  That's where it would be, right?  (That's where I have been grabbing version strings from, as honestly I'm a bit confused by it all, but that seems to work).

Although I thought I'd read something about PIL matrix somewhere too...accoriding to this it is 'fixed' - https://forum.kodi.tv/showthread.php?tid=348400 - but I don't know what version string I should be using..

(I can say the above import works, for now, at least)

Reply:
PIL is a python binary module so it is in kodi source: https://github.com/xbmc/xbmc/pull/17794

Reply:
It's fine as is since kodi addons require a minimum or higher version than x.x.x when you define them as dependencies. Keep it in mind for the future, if you addon works well with the current `7.1.2` and this version is available, it is a good practice to lock the import to that specific module. That way we know which version is supposed to work or not :)